src/opcodes/opcodes.cairo

```diff
@@ -174,6 +188,16 @@ pub mod Opcode {
         engine.dstack.push_int(a + b);
     }
 
+    fn opcode_greaterthan(ref engine: Engine) {
+        let a = engine.dstack.pop_int();
+        let b = engine.dstack.pop_int();
+        engine.dstack.push_int(if a > b {
```

Comment:
I believe this should returning 1 if b > a, how it is now if a == b you will get true. Might want to double check this

Reply:
Okay I will work on this now

Reply:
I think something like
```
if b > a {
    1
} else {
    0
}
```
Would be more efficient here.

Reply:
Okay cool, will use that instead